server/src/main/java/org/opensearch/tasks/CancellableTask.java

```diff
@@ -74,6 +82,8 @@ public CancellableTask(
     public void cancel(String reason) {
         assert reason != null;
         if (cancelled.compareAndSet(false, true)) {
+            this.cancelledAt = System.currentTimeMillis();
+            this.cancelledAtNanos = System.nanoTime();
```

Comment:
How about cancellationStartTime

Reply:
I had kept cancellationStartTime initially but changed this to cancelledAt as per @dblock comments - https://github.com/opensearch-project/OpenSearch/pull/7445/files/3994b25508ac732ab1bcd1729e5de6e114ec566a#r1195505867.
I would have preferred cancellationStartTime as well but don't have a strong opinion as such on this. Let me know what you think.

Reply:
start_time gives an indication that it would be used for computation of elapsed time based on logical time.

Reply:
I agree on that. Lets keep it cancellationStartTime finally then @dblock? As don't want to go and fro on this.

Reply:
It's an internal variable so I don't think this matters that much, A-OK by me.

Reply:
Changed back to `cancellationStartTime` @Bukhtawar